go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/Unknwon/com
+
+require (
+	github.com/gopherjs/gopherjs v0.0.0-20181103185306-d547d1d9531e // indirect
```

Comment:
This looks strange to me, I think this package does not dependent on gopherjs at all?

Reply:
It's a transitive dependency, here's the full path:

```
$ go mod why -m github.com/gopherjs/gopherjs
```
```
# github.com/gopherjs/gopherjs
github.com/Unknwon/com
github.com/Unknwon/com.test
github.com/smartystreets/goconvey/convey
github.com/jtolds/gls
github.com/gopherjs/gopherjs/js
```